api/scpca_portal/models/project.py

```diff
@@ -892,20 +889,22 @@ def load_libraries_metadata(self, samples_metadata: List[Dict]):
                 list(Path(sample_dir).glob("*_metadata.json"))
                 + list(Path(sample_dir).rglob("*_spatial/*_metadata.json"))
             )
```

Comment:
What if instead of only creating one variable here we created two one for single_cell_library_metadata and another for spatial. Then instead of checking a random key in the dict we could just check which list the current one came from is in? We could also attach modality here.

Reply:
Love this suggestion. Added in.